.lighthouserc.js

```diff
@@ -0,0 +1,28 @@
+const baseUrl = 'http://localhost:5601';
```

Comment:
baseUrl technically doesn't have to be 5601. It can be configurable to be something like 443

Reply:
it would be good to pass this as a variable